gulp/tasks/test.js

```diff
@@ -11,10 +12,15 @@ var deps = require('../deps')(config);
 
 var isTestDebug = util.env.d || util.env.debug;
 var testRequirements = isTestDebug ? [] : ['buildTest'];
+var itemInChunk = util.env['items-in-chunk'] || 5;
+
+// TODO: Remove after update lodash to v3.10.1 or higher.
+_.chunk = require('../../node_modules/karma/node_modules/lodash/array/chunk.js');
```

Comment:
Are there some problems with updating upto new version of lodash? Why don't do it now?